model/src/main/kotlin/utils/OrtResultExtensions.kt

```diff
@@ -84,3 +90,36 @@ fun OrtResult.getDetectedLicensesWithCopyrights(
     omitExcluded: Boolean = true
 ): Map<String, Set<String>> =
     LicenseResolver(this, packageConfigurationProvider).getDetectedLicensesWithCopyrights(id, omitExcluded)
+
+/**
+ * Create [PackageConfiguration]s for the projects in this [OrtResult]. [PathExclude]s and [LicenseFindingCuration]s are
+ * taken from the [RepositoryConfiguration]. The [Provenance] is taken from the scan results. Projects without scan
+ * result and scan results without provenance information are ignored.
+ */
+fun OrtResult.getProjectConfigurations(): List<PackageConfiguration> {
+    fun VcsInfo.toMatcher() = VcsMatcher(
+        type = type,
+        url = url,
+        revision = resolvedRevision ?: revision,
```

Comment:
I believe the fallback to revision should not be made. There should always be a resolved revision and package configrations do require it to be (not a branch name on anything moving).